<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-CFSKSBJ" failures="2" tests="4" name="Test" time="8.239" errors="0" timestamp="2024-10-19T18:38:02 IST">
  <testcase classname="api.test.UserTests2" name="testPostUser" time="4.349"/>
  <testcase classname="api.test.UserTests2" name="testGetUserByName" time="1.177">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at api.test.UserTests2.testGetUserByName(UserTests2.java:53)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 36 stack frames]]>
    </failure>
  </testcase> <!-- testGetUserByName -->
  <testcase classname="api.test.UserTests2" name="testUpdateUserByName" time="1.115">
    <failure type="java.lang.AssertionError" message="expected [200] but found [405]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [405]
at api.test.UserTests2.testUpdateUserByName(UserTests2.java:68)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 36 stack frames]]>
    </failure>
  </testcase> <!-- testUpdateUserByName -->
  <testcase classname="api.test.UserTests2" name="testDeleteUserByName" time="1.131"/>
</testsuite> <!-- Test -->
